### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by roesel**
Rough version. Might need refining.

### CODE REVIEW

**Comment by olivierlacan** [comment]
Thank you! Asked for reviews from my Twitter followers. If you know anybody else who speaks Czech and could review it, that'd be great. 😃

**Comment by roesel** [comment]
Still no review? I am not sure why I even bothered :D.

**Comment by olivierlacan** [comment]
@roesel I doubt I have many Czech followers. Can you go over your rough versions and make sure everything's good?

File: source/cs/0.3.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  `NEWS.txt`, `NEWS.md`, `News.txt`, `RELEASES.txt`, `RELEASE.md`, 
+  `releases.md`, apod.
+
+  Je v tom binec. Všecha tato jména jen komplikují život lidem, kteří se ten 
+  dokument snaží najít. 
+  
+  ### Proč lidé jednoduše nepoužívají `git log` diff?
+  Protože diffy logů jsou plné šumu — přirozeně. Nebyly by vyhovujícím 
+  changelogem ani v hypotetickém projektu vedeném dokonalými lidmi, kteří nikdy
+  nedělají překlepy, nikdy nezapomínají commitnout nové doubor a nikdy jim
```
Comment (ondrejtichacek) [inline source/cs/0.3.0/index.html.haml]: Typo in the line 107; it should state: `nedělají překlepy, nikdy nezapomínají commitnout nové soubory a nikdy jim`

Comment (roesel) [inline source/cs/0.3.0/index.html.haml]: Thanks for the review. I agree with this change, it's a silly typo, but I have no idea how to change it without making a new pull request.

**Comment by roesel** [comment]
Thanks for the review. I agree with this change, it's a silly typo, but I have no idea how to change it without making a new pull request.

**Comment by renekliment** [comment]
@roesel When you have stuff that you want to add to a PR, just commit in the branch you want to merge from. In this case it is your `master` - you should always create feature branches for PRs though.

**Comment by zapashcanon** [comment]
@roesel, correct the typo, then:
```sh
git add . && git commit --amend --no-edit && git push -f
```

**Comment by olivierlacan** [comment]
@roesel Can you update the PR and amend? I'd love to publish this. Thanks. 😃

**Comment by roesel** [comment]
@olivierlacan Not quite ammended, but at least fixed in another commit.

**Comment by olivierlacan** [comment]
@roesel Thanks for your work. :-)

**Pull request merged**